nifi-registry/nifi-registry-core/nifi-registry-security-api/src/main/java/org/apache/nifi/registry/security/authorization/UserGroupProvider.java

```diff
@@ -75,6 +75,30 @@ public interface UserGroupProvider {
      */
     Group getGroup(String identifier) throws AuthorizationAccessException;
 
+    /**
+     * Retrieves a Group by name.
+     *
+     * @param name the name of the group to retrieve
+     * @return the Group with the given name, or null if no matching group was found
+     * @throws AuthorizationAccessException if there was an unexpected error performing the operation
+     */
+    default Group getGroupByName(String name) throws AuthorizationAccessException {
```

Comment:
As this method appears to be used only in the `StandardManagedAuthorizer`, it does not seem like a good candidate for adding to the `UserGroupProvider` interface right now.

Reply:
Ahh, right. That was a changed I pulled from the nifi major OIDC rewrite but I agree it doesn't make sense to do here at this point. I've removed this method from the interface and moved the logic into the `StandardManagedAuthorizer`.